src/System.Private.CoreLib/shared/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -2263,12 +2265,9 @@ internal TokenHashValue[] CreateTokenHashTable()
                     InsertHash(temp, dateSeparatorOrTimeZoneOffset, TokenType.SEP_DateOrOffset, 0);
                 }
 
-                string[] dateWords = null;
-                DateTimeFormatInfoScanner scanner = null;
-
                 // We need to rescan the date words since we're always synthetic
-                scanner = new DateTimeFormatInfoScanner();
-                dateWords = scanner.GetDateWordsOfDTFI(this);
+                DateTimeFormatInfoScanner scanner = new DateTimeFormatInfoScanner();
+                string[]? dateWords = dateWords = scanner.GetDateWordsOfDTFI(this);
```

Comment:
Double initializing, doesn’t seem intentional

Reply:
lol - I had to look like 3 times at this line to spot it, my eyes just skipped through the mid section - good catch!